jest.config.js

```diff
@@ -76,5 +76,9 @@ module.exports = {
       "<rootDir>/../out/Release/gen/ui/webui/resources/preprocessed/$1",
       "<rootDir>/../out/Debug/gen/ui/webui/resources/preprocessed/$1",
     ]
-  }
+  },
+  reporters: [
+      "default",
+      ["jest-junit", { "outputName": "brave_jest_tests.xml" }]
```

Comment:
Do we need to selectively enable this only in CI via an env variable and/or exclude this file in .gitignore?